backend/history-service/src/middleware/auth.ts

```diff
@@ -37,10 +37,8 @@ export const authMiddleware = async (
   }
 
   //If there is JWT, validate it through the auth endpoint
-  const NODE_ENV = process.env.NODE_ENV || "development";
   const authEndpoint =
-    process.env.AUTH_ENDPOINT ||
-    `http://localhost:5050/api/${NODE_ENV}/auth/api/auth/validate`;
+    process.env.AUTH_ENDPOINT || `http://localhost:5050/auth/api/validate`;
```

Comment:
What will be the value of `${AUTH_GATEWAY}`? But anyway for local dev, this is the working version, we can update this again in the future, in a separate PR.

Reply:
${AUTH_GATEWAY} will be the dnsname of the auth service, like `http://auth-service`. it will be defined in the aws task definition.

Reply:
I see! However, I think I can't update the url now because it will break existing setup in our local test. Let's update this again in the future PR. Thanks!

Reply:
sure, we can update this when we are ready to deploy history into the cloud